run/test/specs/user_actions_change_profile_picture.spec.ts

```diff
@@ -44,15 +45,15 @@ async function changeProfilePictureAndroid(platform: SupportedPlatformsType) {
     maxWait: 10000,
   });
   // Waiting for the image to change in the UI
-  await sleepFor(1000);
+  await sleepFor(10000);
   const base64 = await device.getElementScreenshot(el.ELEMENT);
+  fs.writeFileSync('screenshot.png', base64, 'base64');
```

Comment:
can be removed? If not, make sure to ignore it in gitignore

Reply:
Removed

Reply:
@burtonemily I don't see screenshot.png in the gitignore? Is it still needed? Does screenshot.png get generated automatically at any point?

Reply:
No I just removed the screenshot and the function, it didn't fix the problem I was trying to debug.